golang/server/server.go

```diff
@@ -105,7 +105,7 @@ func (server MinimalGRPCServer) RunUntilStopped(stopper <-chan struct{}) error {
 		registrationFunc(grpcServer)
 	}
 
-	listenAddressStr := fmt.Sprintf(":%v", server.listenPort)
+	listenAddressStr := fmt.Sprintf("0.0.0.0:%v", server.listenPort)
```

Comment:
@Peeeekay I think that we should use `127.0.0.1` instead of `0.0.0.0` for making compatible with Windows

Reply:
0.0.0.0 accepts all hosts and the server is inside the docker container - so should work with windows.